awx/ui/client/test/e2e/settings.js

```diff
@@ -0,0 +1,41 @@
+const AWX_E2E_URL            = process.env.AWX_E2E_URL            || 'https://localhost:8043';
```

Comment:
I'd vote for not vertically aligning anything, but that's just a personal preference.

Reply:
sounds good to me - done